tools/Dockerfile.tools

```diff
@@ -0,0 +1,42 @@
+FROM otomi/base:v1.4.22
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# https://cloud.google.com/sdk/docs/downloads-versioned-archives
+ARG GOOGLE_SDK_VERSION=367.0.0
+# https://github.com/open-policy-agent/opa/releases
+ARG OPA_VERSION=0.33.1
+# https://github.com/zegl/kube-score/releases
+ARG KUBESCORE_VERSION=1.12.0
+
+USER root
+
+RUN apt-get update -qqy && apt-get install -qqy --no-install-recommends && \
+  unzip \
+  && rm -rf /var/lib/apt/lists/*
+
+# aws
+ADD https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip awscliv2.zip
+RUN unzip awscliv2.zip && aws/install -i awscli -b awscli/bin && rm -rf aws awscliv2.zip
+
+# aws-iam-authenticator
+ADD https://amazon-eks.s3.us-west-2.amazonaws.com/1.19.6/2021-01-05/bin/linux/amd64/aws-iam-authenticator aws-iam-authenticator
+RUN chmod +x aws-iam-authenticator
+
+# gcloud
+ADD https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz /tmp/gsdk.tar.gz
+RUN tar -zxvf /tmp/gsdk.tar.gz -C /tmp && mv /tmp/google-cloud-sdk gsdk && rm -rf /tmp/*
+
+# opa
+ADD https://github.com/open-policy-agent/opa/releases/download/v${OPA_VERSION}/opa_linux_amd64 opa
+RUN chmod +x opa
+
+# kubescore
+ADD https://github.com/zegl/kube-score/releases/download/v${KUBESCORE_VERSION}/kube-score_${KUBESCORE_VERSION}_linux_amd64 kube-score
+RUN chmod +x kube-score
+
+RUN chown -R app:app $TOOLS_HOME
```

Comment:
Double quote to prevent globbing and word splitting: https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
Not needed as no spaces in path. We do DRY